### PR TITLE
TASK: Add PHP 7.0 scalar type hints to method arguments and return types

### PR DESCRIPTION
**Pull request by albe**

### CODE REVIEW

**Comment by albe** [comment]
Yep, was to be expected somehow with ObjectAccess though :) I will take care ASAP you fix things

File: Neos.Eel/Classes/FlowQuery/Operations/Object/PropertyOperation.php [hunk 1]
```diff
@@ -49,17 +49,17 @@ class PropertyOperation extends AbstractOperation
      */
     public function evaluate(FlowQuery $flowQuery, array $arguments)
     {
-        if (!isset($arguments[0]) || empty($arguments[0])) {
+        if (!isset($arguments[0]) || empty($arguments[0]) || !is_string($arguments[0])) {
```
Comment (albe) [inline Neos.Eel/Classes/FlowQuery/Operations/Object/PropertyOperation.php]: Not 100% sure about this. Can this argument be for example an object that can be `toString()`'d?

Comment (kdambekalns) [inline Neos.Eel/Classes/FlowQuery/Operations/Object/PropertyOperation.php]: It shouldn't be, IMHO.

**Comment by kdambekalns** [comment]
Enough of the waiting… :)

**Pull request merged**